Mobl_ios_Callisto-master/MealTrackingPilot/Navigation/AppNavigator.swift

```diff
@@ -29,7 +28,7 @@ final class AppNavigator: NSObject, LoginClientDelegate, LoginClientUIDelegate {
     }()
 
     private lazy var signInViewController: LoginLandingViewController = {
-        let controller = LoginLandingViewController.ip_fromNib()
+        let controller = LoginLandingViewController.init(nibName: LoginLandingViewController.nibName, bundle: nil)
```

Comment:
Please use here the fromNib() instead of whole init.

Reply:
done changes